api/src/main/java/org/openmrs/scheduler/timer/TimerSchedulerServiceImpl.java

```diff
@@ -542,5 +542,14 @@ public void scheduleIfNotRunning(TaskDefinition taskDef) {
 			}
 		}
 	}
+
+	/**
+	 * @see org.openmrs.scheduler.SchedulerService#getTaskByUuid(java.lang.String uuid)
```

Comment:
What is the use of `uuid` in the above line?

Reply:
i have was trying to indicate the parameter to but i have removed it